content/blog/end-to-end-encryption/index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: End-to-End Encryption in the Browser
+date: '2020-03-21T22:40:32.169Z'
+description: "&nbsp;"
+---
+
+[Excalidraw](https://excalidraw.com/) is a whiteboard tool that lets you easily sketch diagrams that have a hand-drawn feed to them. It is very handy to dump your thoughts many of which are sensitive: designs for new features not yet released, interview questions, org charts...
+
+By default, [Excalidraw](https://excalidraw.com/) is not sending any of the content you are drawing to the server. But sometimes it is useful to be able to send a link to the scene you are working on to someone else.
+
+You could save the file, send it to the other person, have them open it. But that's pretty cumbersome. In this post we'll see how to get a link working without the server having access to the data.
+
+## Traditional Website Architecture
+
+In the traditional website architecture, if you want to save a scene, you're going to send the content to the server so that it can store it. Then to load it you'll the server to send it back to you.
+
+In that world, you trust the server to contain your information but you don't trust the pipes in-between the client and server using https to encrypt the data.
```

Comment:
```suggestion
In that world, you trust the server to contain your information but you don't have to trust the pipes in between the client and the server, because you use HTTPS to encrypt the data.
```